scripts/run_integration_tests.sh

```diff
@@ -36,6 +36,12 @@ cd $integration_tests_dir
 for ((i = 0 ; i < ${#SERVERLESS_CONFIGS[@]} ; i++)); do
     echo "Running 'sls package' with ${SERVERLESS_CONFIGS[i]}"
     serverless package --config ${SERVERLESS_CONFIGS[i]}
+    # Normalize unstable values, perl substitution command: perl -p -i -e 's/(PATTERN TO FIND)/PATTERN TO SUBSTITUTE/g'
+    perl -p -i -e 's/("S3Key".*)/"S3Key": "serverless\/dd-sls-plugin-integration-test\/dev\/XXXXXXXXXXXXX-XXXX-XX-XXXXX:XX:XX.XXXX\/dd-sls-plugin-integration-test.zip"/g' .serverless/cloudformation-template-update-stack.json
+    perl -p -i -e 's/(LambdaVersion.*")/LambdaVersionXXXX"/g' .serverless/cloudformation-template-update-stack.json
+    perl -p -i -e 's/("CodeSha256":.*)/"CodeSha256": "XXXX"/g' .serverless/cloudformation-template-update-stack.json
+    perl -p -i -e 's/(v\d.\d\d.\d)/vX.XX.X/g' .serverless/cloudformation-template-update-stack.json
```

Comment:
Nit: Can we find a way to rewrite this to not repeat `.serverless/cloudformation-template-update-stack.json` on every line?